processor/src/operations/utils.rs

```diff
@@ -1,7 +1,7 @@
 use super::{ExecutionError, Felt};
 use crate::{ErrorContext, ONE, ZERO};
 
-/// TODO: add docs
+/// Asserts that the given value is a binary value (0 or 1).
 #[inline(always)]
 pub fn assert_binary_with_ctx(
```

Comment:
should this be named just `assert_binary`?

Reply:
Good point - fixed